lib/rules/prefer-const.js

```diff
@@ -82,7 +82,7 @@ function canBecomeVariableDeclaration(identifier) {
  *      Otherwise, null.
  */
 function getIdentifierIfShouldBeConst(variable, ignoreReadBeforeAssign) {
-    if (variable.eslintUsed) {
+    if (variable.eslintUsed && variable.scope.type === "global") {
```

Comment:
Can you add a comment in the JSDoc to explain why you're checking for global scope?